reports/example.md

```diff
@@ -8,36 +8,46 @@ organisation: coala
 organisation_link : https://coala.io
 project: <project title>
 project_link: <project link>
+report_year: <year of completing GSoC>
 tarball: <tarball of patches' link>
+sha256sum: <sha256sum of tarball>
 mentors: >
- [mentor1]<link1> & [mentor2]<link2>
+  [mentor1]<link1> & [mentor2]<link2>
 phase:
- - Bonding : <bonding milstone link>
- - Phase 1 : <phase1 milsetone link>
- - Phase 2 : <phase2 milsetone link>
- - Phase 3 : <phase3 milsetone link>
+  - Bonding : <bonding milstone link>
```

Comment:
`milstone` was a typo, and you copied that without detecting typo ;)

Reply:
Ah, it's beyond me how you guys can actually figure out a typo in the whole english mess. ':D